app/build.gradle

```diff
@@ -5,6 +5,10 @@ plugins {
     id 'dagger.hilt.android.plugin'
 }
 
+plugins {
+    id 'com.onesignal.androidsdk.onesignal-gradle-plugin'
```

Comment:
lets just put this in the plugins section we have above instead of making a new plugins section